### PR TITLE
Add more examples for IOS

### PR DESCRIPTION
**Pull request by thaixuann**

### CODE REVIEW

File: clean/full_clean_C3560CX_ios/c3560cx_clean.yaml [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+      origin:
+        files:
+          - mypath/test.cfg
+        hostname: 10.15.19.10
+      destination:
+        directory: 'flash:/'
+      protocol: ftp
+      check_file_stability: True
+      min_free_space_percent: 50
+      # if_exist_still_copy: True
```
Comment (d-the-iii) [inline clean/full_clean_C3560CX_ios/c3560cx_clean.yaml:29]: I would recommend taking this out or changing it to use the actual flag, `overwrite: True`

File: clean/full_clean_ie4000_ios/ie4000_clean.yaml [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    copy_to_device:
+      origin:
+        files:
+          - mypath/test.cfg
+        hostname: 10.15.19.10
+      destination:
+        directory: 'flash:/'
+      protocol: ftp
+      min_free_space_percent: 50
+      # if_exist_still_copy: True
```
Comment (d-the-iii) [inline clean/full_clean_ie4000_ios/ie4000_clean.yaml:28]: same comment here

File: clean/full_clean_ie5000_ios/ie5000_clean.yaml [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    copy_to_device:
+        origin:
+            files:
+                - mypath/test.cfg
+            hostname: 10.15.19.10
+        destination:
+            directory: 'flash:/'
+        protocol: ftp
+        min_free_space_percent: 10
+        # if_exist_still_copy: True
```
Comment (d-the-iii) [inline clean/full_clean_ie5000_ios/ie5000_clean.yaml:28]: same

**Review by d-the-iii — APPROVED** [review]
LGTM

**Review by omehrabi — APPROVED** [review]

**Pull request merged**